mountpoint-s3-crt-sys/UPDATING_CRT.md

```diff
@@ -34,3 +39,16 @@ In order to check which released versions are checked out for each submodules, t
 ```sh
 git submodule foreach -q 'echo $name `git describe --tags`'
 ```
+
+## Crate size
+
+As the AWS CRT project evolves, the size of the `mountpoint-s3-crt-sys` crate can grow or shrink.
+We manage its C-based dependencies with Git submodules, which can include all sorts of files we don't control.
+
+Inside the cargo manifest for that package,
+you can find an `excludes` entry which lists a number of patterns for excluding files
+from the compressed archive uploaded to [crates.io](https://crates.io/).
+We exclude files that we know won't be used, like files within `.github/` paths or PDFs.
+
+If the crate grows too large, we can add new patterns to this list.
+You can view the files included in the archive using `cargo package -p mountpoint-s3-crt-sys --list`.
```

Comment:
Worth also adding the command to check the crate size? I think:
```
cargo publish --package mountpoint-s3-crt-sys --dry-run
```

Reply:
Yeah, good idea.